python/ray/autoscaler/aws/config.py

```diff
@@ -37,13 +37,18 @@
     "Boto3 version >= 1.4.8 required, try `pip install -U boto3`"
 
 
-def key_pair(i, region):
+def key_pair(i, region, key_name):
```

Comment:
Can you please document? 

Also, don't we already support custom KeyPairs?

Reply:
Added an one-liner for `key_name` argument.

As for custom key pair, my understanding is that what we support today is accepting user's **existing** key pairs. I want to add the ability to **create a new key pair with custom names**. We previously achieve this by creating a key pair with a custom name ourselves and passing it in to autoscaler, but then we have some code duplication with autoscaler (namely `_configure_key_pair`)

Reply:
Got it; thanks!